cypress/support/pages/course/pageCommonCourses.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+class Courses_Common {
+
+  static #learningContentTab = '[data-testid="learnContent"]'
+  static #toolsTab = '[data-testid="tools"]'
+  static #groupsTab = '[data-testid="groups"]'
+  static #toolsList = '[data-testid="course_tool_list_add_tool"]'
+  static #pageTitle = '[id="page-title"]'
+  static #courseOverviewNavigationButton = '[data-testid="Course-Overview"]'
+  static #addNewToolButton = '[data-testid="add_new_tool"]'
+
+  navigateToRoomsOverview() {
+    cy.get(Courses_Common.#courseOverviewNavigationButton).click()
+    cy.url().should('include', '/rooms-overview')
+  }
+
+  navigateToRoomBoard(room_name) {
+    const selectedRoom = `[aria-label='${room_name}']`
```

Comment:
it should be outside of the function

Reply:
this will be changed in a follow up then, currently it is like this on main as well.